src/molecule.py

```diff
@@ -2891,28 +2868,11 @@ def read_xyz0(self, fnm, **kwargs):
             if ln == 0:
                 # Skip blank lines.
                 if len(line.strip()) > 0:
-                    try:
```

Comment:
The lines highlighted in red are newer, so the changes here need to be reverted, perhaps by merging the master branch into this branch.